packages/renderers-rust/test/_setup.ts

```diff
@@ -10,3 +10,14 @@ export function codeContains(actual: string, expected: (RegExp | string)[] | Reg
         }
     });
 }
+
+export function codeNotContains(actual: string, expected: (RegExp | string)[] | RegExp | string) {
```

Comment:
nit: I called this `codeDoesNotContain` on the JS side IIRC. Should we keep the same name to stay consistent?